core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -160,7 +160,7 @@ object UserService {
     for {
       petSAJson <- samDAO.getPetServiceAccountKeyForUser(projectName, RawlsUserEmail(userIdInfo.userEmail))
```

Comment:
There is quite a bit of duplicated code here between `UserService` and `WorkspaceService`, but supposedly it can all be deleted once workspace migration is complete (https://broadworkbench.atlassian.net/browse/CA-1118).